include/envoy/config/grpc_mux.h

```diff
@@ -113,6 +113,13 @@ class GrpcMux {
    */
   virtual void resume(const std::string& type_url) PURE;
 
+  // TODO(fredlas) PR #8478 will remove this.
+  /**
+   * Whether this GrpcMux is delta.
```

Comment:
What's the end goal for supporting a combination of SOTW and delta? I would imagine that some operators will want both, with let's say CDS delta but LDS SOTW. Can that be done over a single gRPC stream?

Reply:
Yes and no and maybe eventually yes, haha. Non-aggregated xDS can already mix and match delta and SotW no problem. When using ADS, because delta and SotW have different request/response protos, and different gRPC services corresponding to those protos, there can only be one type.

However, the change we briefly discussed a while back, where there would be a single top level carrier proto for request+response, and the various types of messages (delta vs SotW, non-ACK request vs ACK) were just different oneof options, would enable mixing within ADS. In practice that would take a bit of plumbing in this code (probably pretty straightforward), and would also need the bootstrap config format to be reworked a bit: rather than just configuring your CDS as "CDS, provided by ADS [implicitly referring to the ADS config]", you would also need to choose delta/SotW in the CDS config thing. And... actually, the choice would be removed from the ADS config thing.

Reply:
Ack. OK, let's go ahead with this as is, thanks for the reminder of our earlier conversation on this.